src/components/Footer.astro

```diff
@@ -16,9 +16,10 @@ import Icon from "./Icon.astro"
 
 const navigation = {
   support: [
-    { name: "Documentation", href: DOCUMENTATION_URL },
-    { name: "GitHub Issues", href: GITHUB_ISSUES_URL },
+    { name: "Docs", href: DOCUMENTATION_URL },
+    { name: "Issues", href: GITHUB_ISSUES_URL },
     { name: "Discord", href: DISCORD_URL },
+    { name: "Email", mailto: "support (at) ddev.com" }
```

Comment:
`mailto` isn’t a convention here, so you’ll see that this link doesn’t do anything. Even if it was, I’d be annoyed to have it open an email client and populate the `to:` field with an invalid address—wouldn’t you?

Rather than add links here, it might be more effective to create a catch-all Support page and link to that instead—like we do with the “Support DDEV” page as a clear call to action.

More links are more decision points and ultimately more noise, so it’s worth thinking through funnels and reinforcing them.

Reply:
Please let me know what you'd do here to just give away the email address without giving it away to the world.

Reply:
My experience has been that I’ll collect spam regardless of anything I could possibly do, and it [seems like](https://stackoverflow.com/questions/11563283/why-write-at-and-dot-in-email-rather-than-and) the [jury’s out](https://www.quora.com/Is-writing-out-the-at-in-email-addresses-effective-to-prevent-spam?share=1) on “mailbox (at) domain” formatting being effective.

I’d recommend any kind of obfuscation that isn’t a penalty to the visitor. (Because `mailto:support (at) ddev.com` would put the burden on me to fix it in my email client in order to send mail to `support@ddev.com`.)

[This article](https://spencermortensen.com/articles/email-obfuscation/) suggests that either URL encoding or CSS would be viable approaches.

Reply:
Changed to use URL encoding, thanks so much!

Reply:
I think a contact form shouldn't be hard and should be in our future. Will leave it with the obfuscation right now.